third-party/rspamd/README.md

```diff
@@ -0,0 +1,40 @@
+## Additional webadmin endpoints
+
+### Report spam messages to RSpamD
+One can use this route to schedule a task that reports spam messages to RSpamD for its spam classify learning.
+
+```bash
+curl -XPOST 'http://ip:port/rspamd?action=reportSpam
+```
+
+This endpoint has the following param:
+- `action` (required): need to be `reportSpam`
+- `messagesPerSecond` (optional): Concurrent learns performed for RSpamD, default to 10
+- `period` (optional): duration (in seconds), only messages between `now` and `now - duration` are reported. By default, 
+all messages are reported.
+- `samplingProbability` (optional): float between 0 and 1, represent the chance to report each given message to RSpamD. 
+By default, all messages are reported.
+
+Will return the task id. E.g:
+```
+{
+    "taskId": "70c12761-ab86-4321-bb6f-fde99e2f74b0"
+}
+```
```

Comment:
Link to https://james.apache.org/server/manage-webadmin.html#Endpoints_returning_a_task instead (as endpoints returning tasks is something generic, you do not need to detail it here).